parsl/executors/high_throughput/executor.py

```diff
@@ -264,6 +265,7 @@ def initialize_scaling(self):
 
         self._scaling_enabled = True
         logger.debug("Starting HighThroughputExecutor with provider:\n%s", self.provider)
+        # TODO: why is this a provider property?
```

Comment:
prototyping history more than anything, I think

Reply:
This comment could be removed at this point.